### PR TITLE
{bio}[foss/2018a] FLASH v1.2.11

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2610.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/9a420de7d7beaaf77fe86b4f6783349d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3161.skitty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/e2bdacb8daa531a14c88dd9d1c5612b7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2109.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/6dd9aa865a845d1cc59d2ed4d3a8b8a1 for a full test report.

**Comment by vanzod** [comment]
Test report by @vanzod
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 9.4, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/90e98a9a97343d705763cd5088159083 for a full test report.

File: easybuild/easyconfigs/f/FLASH/FLASH-1.2.11-foss-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+description = """FLASH (Fast Length Adjustment of SHort reads) is a very fast and accurate software
+  tool to merge paired-end reads from next-generation sequencing experiments. FLASH is designed to
+  merge pairs of reads when the original DNA fragments are shorter than twice the length of reads.
+  The resulting longer reads can significantly improve genome assemblies. They can also improve
+  transcriptome assembly when FLASH is used to merge RNA-seq data."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['http://download.sourceforge.net/%(namelower)spage']
+sources = [SOURCE_TAR_GZ]
```
Comment (vanzod) [inline easybuild/easyconfigs/f/FLASH/FLASH-1.2.11-foss-2018a.eb]: Missing checksum

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2007.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/cc14e55c777e2004e8ec8535bb5dad14 for a full test report.

**Comment by vanzod** [comment]
Test report by @vanzod
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 9.4, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/2f5dbf2637f1726b181091b4b6985f3d for a full test report.

**Comment by vanzod** [comment]
Going in, thanks @boegel!

**Pull request merged**